src/lib/Components/ArtworkGrids/Artwork.tsx

```diff
@@ -75,14 +75,14 @@ class Artwork extends React.Component<Props, any> {
         <Badges>
           {is_acquireable && (
             <Badge>
-              <Sans weight="medium" size="1">
+              <Sans weight="medium" fontSize={8} lineHeight={9} size="1">
```

Comment:
`lineHeight` here needs to be manually set (and 1 higher than the font size). On web this can be set to `"1"` (which means it will equal the font size).

Reply:
We shouldn't be defining custom font sizes or line heights as that breaks our design system. It looks like the closest size is 10px, which is `size=1` (https://github.com/artsy/palette/blob/master/src/Theme.tsx#L148). 

@briansw - thoughts? These sort of 1-offs lead to design drift.

Reply:
Yep, it's not meant as a one off. We mentioned in the design system sync meeting last week that we're working on paring down some of our type sizes. That's still in flux, but this size is part of that adjustment. I'm all for adding a `size 0` to palette in the interim. cc @zephraph

Reply:
Updated here: https://github.com/artsy/palette/pull/102